src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -194,7 +193,11 @@ public void testUpdateFeedbackResponseComment() throws Exception {
         ______TS("fail: invalid params");
         
         frComment.courseId = "invalid course name";
-        verifyExceptionThrownWhenUpdateFrComment(frComment, "not acceptable to TEAMMATES as a/an course ID");
+        String expectedError = getPopulatedErrorMessage(
```

Comment:
Not sure what @damithc thinks about this, ```"not acceptable to TEAMMATES as a/an course ID"``` is basically the constant ```FieldValidator.COURSE_ID_ERROR_MESSAGE``` so I don't see a problem since the frontend probably uses that ```FieldValidator.COURSE_ID_ERROR_MESSAGE``` to generate it's message anyway